pages/algorithms/exposed-to-assistive-technologies.md

```diff
@@ -9,3 +9,6 @@ Elements contained in the accessibility tree as described in one of the followin
 - [HTML Accessibility API Mappings](https://www.w3.org/TR/html-aam)
 - [SVG Accessibility API Mappings](https://www.w3.org/TR/svg-aam)
 
+Elements that are included in the accessibility tree are exposed to assistive technologies. 
+
+The elements that are included in the accessibility tree are however not the only ones that might be exposed to assistive technologies. An example is that elements that are [focusable](#focusable) can still be reached by a screen reader through tabbing.
```

Comment:
The file will need to be renamed as well.

Reply:
This strongly implies that this definition shouldn't be renamed. I'd prefer that, as the accessibility tree is a far more technical term than assistive technologies.